oemer/bbox.py

```diff
@@ -2,9 +2,14 @@
 import cv2
 import numpy as np
 from sklearn.cluster import AgglomerativeClustering
+from numpy import ndarray
+from typing import List
+from typing import Tuple
+from typing import Union, Any
+from numpy import float64
 
 
-def get_bbox(data):
+def get_bbox(data: ndarray) -> List[Tuple[int, int, int, int]]:
```

Comment:
You could define a type specifically for BBox like `BBOX_TYPE = Tuple[int, int, int, int]` and annotated the return type to `-> List[BBOX_TYPE]`. It can also be reused in other places like the type of function variables `def foo(bbox: BBOX_TYPE)`.